src/main.rs

```diff
@@ -81,24 +81,29 @@ impl PayloadItem {
 
         payload_item.unwrap()
     }
+
+    // If the value is defined as systemd.time, converts the defined duration into a UNIX timestamp
+    fn process_timestamp(val: Option<&str>, name: &str, now: i64) -> Option<PayloadItem> {
```

Comment:
Since this isn't an instance method, let's rename this to `from_timestamp_with_name` to match `from_string_from_name` above

Reply:
Renamed!